### ISSUE TITLE
Add a date to each notebook 

### ISSUE THREAD

**Issue by emiliom** [ioos/ioos_code_lab#32]
The new JupyterBook-based IOOS Code Lab looks great!

But one thing that I suggest should be added to each notebook (which was present in the previous IOOS Code Gallery, I think) is a date, making it clear to users when the notebook was developed. That's a useful piece of information to help decide if the code used in the notebook might be old and the packages used may have newer, better features.

Currently, a sharp user will notice that the date is in the notebook file name, in the url. But it'd be better to make that information less subtle. It could go right below the title or at the end next to the Author line.

**Comment by MathewBiddle** [comment]
@ocefpaf we could probably write a simple script to extract the date from the name of the notebook and insert it as a markdown cell. Just wondering if that's the right approach? 

Would it be easier/more efficient to do it by hand? There are 34 notebooks to update as of now...

**Comment by ocefpaf** [comment]
> @ocefpaf we could probably write a simple script to extract the date from the name of the notebook and insert it as a markdown cell. Just wondering if that's the right approach?
> 
> Would it be easier/more efficient to do it by hand? There are 34 notebooks to update as of now...

Rewriting a cell into an existing notebook is not too hard but I never got it to behave as I would like. Usually it generates more changes than it is desirable. Also, we don't want to re-run those notebooks.

TL;DR maybe we can just bite the bullet and add those manually.
